### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by sachitsac** [amsul/pickadate.js#581]
Hi,

I have a feature request. I am not sure how far the accessibility is supported so depending on the scope, it may be a bug.

I found an issue with accessibility. When i am using the left and right arrow keys to navigate between dates, it doesn't speak out the date. Its just a blank beep. I found another datapicker that does this really well. http://freqdec.github.io/datePicker/

Can we have something similar ? 

Cheers.

**Comment by franktroia** [comment]
Hi, I don't think this issue is resolved yet.

You can see this stackoverflow issue raised for JAWS.
http://stackoverflow.com/questions/28989500/how-do-i-get-pickadate-js-working-with-the-jaws-screenreader

I don't have JAWS, but I've tested on both NVDA and ChromeVox and I can confirm the date is not announced for these readers either.

With NVDA (and maybe JAWS too), in fact, you cannot even use the keyboard since the arrow keys are tied to something NVDA specific. With ChromeVox, the arrow keys work, but the dates are not announced.

**Comment by jasonday** [comment]
Additionally, I am also getting limited ARIA for pickadate.js. Data selection is not announced, and the selected value is not announced either (NVDA).

**Comment by kisin** [comment]
i can confirm that it not working on NVDA - the date is not spoken

**Comment by kisin** [comment]
in the end i solved this problem by mimicking a11yfy script and created an aria-live="assertive" div with the currect date inside it, this solution forcing NVDA reading the date.

not a perfect solution but its working for now.
